phpseclib/Crypt/TripleDES.php

```diff
@@ -258,6 +258,7 @@ public function setKey($key)
         switch (strlen($key)) {
             case 16:
                 $key.= substr($key, 0, 8);
+                break;
             case 24:
                 break;
             default:
```

Comment:
There is no need for this change. Before the change we have this:

```php
        switch (strlen($key)) {
            case 16:
                $key.= substr($key, 0, 8);
            case 24:
                break;
```

After the change we have this:

```php
        switch (strlen($key)) {
            case 16:
                $key.= substr($key, 0, 8);
                break;
            case 24:
                break;
```

They do the same thing. When `strlen($key) == 16` `$key` will be modified and then fallthrough to the `break` for `strlen($key) == 24` case.

https://blog.mariusschulz.com/2015/06/25/the-curious-case-of-switch-statements-in-javascript elaborates. I suppose, as that article suggests, putting `// fallthrough` in comments might not be a bad idea but, in this case, doing `break;` would almost be preferable to that because, despite being redundant, it takes up less bytes. I suppose being concerned about bytes on modern computers is a bit silly tho idk lol